OpenRA.Game/OpenRA.Game.csproj

```diff
@@ -7,7 +7,7 @@
     <PackageReference Include="System.Runtime.Loader" Version="4.3.0" />
   </ItemGroup>
   <ItemGroup>
-    <PackageReference Include="Fluent.Net" Version="1.0.52" />
+    <PackageReference Include="Linguini.Bundle" Version="0.2.0" />
```

Comment:
Update
```suggestion
    <PackageReference Include="Linguini.Bundle" Version="0.2.1" />
```

Reply:
Updated, but it seems to have no effect.